crates/mysten-network/src/multiaddr.rs

```diff
@@ -101,6 +101,17 @@ impl Multiaddr {
         Ok(SocketAddr::new(ip, tcp_port))
     }
 
+    // Returns Ok(true) if the Multiaddr is a valid TCP address.
+    pub fn is_tcp_addr(&self) -> Result<bool> {
```

Comment:
would it make more sense for this api to just return a bool and false in the instance where the multiaddr is short?

Reply:
no strong opinions from me. would you like me to make the change?

Reply:
I think it would make a little bit more sense but i can leave that to you.